bevy_rapier2d/examples/debugdump2.rs

```diff
@@ -0,0 +1,61 @@
+use bevy::prelude::*;
+use bevy_rapier2d::prelude::*;
+
+fn main() {
+    let mut app = App::new();
+    app.insert_resource(ClearColor(Color::srgb(
+        0xF9 as f32 / 255.0,
+        0xF9 as f32 / 255.0,
+        0xFF as f32 / 255.0,
+    )))
+    .add_plugins((
+        DefaultPlugins,
+        RapierPhysicsPlugin::<NoUserData>::pixels_per_meter(100.0),
+        RapierDebugRenderPlugin::default(),
+    ))
+    .add_systems(Startup, (setup_graphics, setup_physics))
+    .add_systems(PostUpdate, display_events);
+
+    bevy_mod_debugdump::print_schedule_graph(&mut app, PostUpdate);
+}
+
+pub fn setup_graphics(mut commands: Commands) {
+    commands.spawn(Camera2dBundle::default());
+}
+
+pub fn display_events(
+    mut collision_events: EventReader<CollisionEvent>,
+    mut contact_force_events: EventReader<ContactForceEvent>,
+) {
+    for collision_event in collision_events.read() {
+        println!("Received collision event: {collision_event:?}");
+    }
+
+    for contact_force_event in contact_force_events.read() {
+        println!("Received contact force event: {contact_force_event:?}");
+    }
+}
```

Comment:
This is not strictly needed ; but if we keep it, it should probably be ordered after `step_simulation`

Reply:
Yeah, let’s just stick to the rapier systems here.